projects/media-viewer/src/lib/toolbar/main-toolbar/left-pane/left-pane.component.html

```diff
@@ -20,10 +20,12 @@
         <span data-l10n-id="next_label">Next</span>
       </button>
     </div>
-    <input type="number" id="pageNumber" class="toolbarField pageNumber" title="Page" value="1" size="4" min="1"
```

Comment:
I thought this just needed an aria-label tag for accessbility

Reply:
yep, changing it now. I was trying to use the recommended approach which is to wrap them in labels but doesn't seem to work. Will be changing to aria-label tags now.